pkg/rancher-desktop/tsconfig.json

```diff
@@ -19,14 +19,24 @@
       "~/*": [
         "./*"
       ],
-      "@/*": [
+      "@pkg/*": [
         "./*"
-      ]
+      ],
+      "@shell/core/*": [
+        "../../node_modules/@rancher/shell/core"
+      ],
     },
+    "typeRoots": [
```

Comment:
Does it make sense to use [`extends`](https://www.typescriptlang.org/tsconfig#extends) here?

Reply:
I think that it would make sense but the resulting tsc errors will take more effort to resolve.